scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,151 @@
+        batch_start_time = time.time()
+        
+        cmd = [
+            "python", 
+            "scripts/download_weather_incremental.py",
+            "--source", "openmeteo",
+            "--country", "JP",
+            "--start", start_date,
+            "--end", end_date
+        ]
+        
+        print(f"  Starting download at {datetime.now().strftime('%H:%M:%S')}")
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)